Exiled.Events/EventArgs/Scp096/StartPryingGateEventArgs.cs

```diff
@@ -58,6 +56,9 @@ public StartPryingGateEventArgs(Scp096Role scp096, Player player, PryableDoor ga
         /// </summary>
         public Door Door { get; }
 
+        /// <inheritdoc cref="Door"/>
+        public Gate Gate => Door.Is(out Gate gate) ? gate : null;
+
```

Comment:
Since prying Event will always have a Gate i thinks the property would be better like this 

You will just need to change the ctor to make it target the Gate property and casting it
```suggestion
        /// </summary>
        public Door Door => Gate;

        /// <inheritdoc cref="Door"/>
        public Gate Gate { get; }

```